src/w3c/templates/headers.js

```diff
@@ -43,12 +55,12 @@ export default conf => {
     ` : ""}
     ${conf.showPreviousVersion ? html`
       <dt>Previous version:</dt>
-      <dd><a href='${conf.prevVersion}'>${conf.prevVersion}</a></dd>
+      <dd><a rel='${conf.doRDFa ? "dcterms:replaces" : null}' href='${conf.prevVersion}'>${conf.prevVersion}</a></dd>
     ` : ""}
     ${conf.prevRecURI ? html`
       ${conf.isRec ? html`
           <dt>Previous Recommendation:</dt>
-          <dd><a href='${conf.prevRecURI}'>${conf.prevRecURI}</a></dd>
+          <dd><a rel='${conf.doRDFa ? "dcterms:replaces" : null}' href='${conf.prevRecURI}'>${conf.prevRecURI}</a></dd>
```

Comment:
You may want to remove these `rel` and `property` things, as the main branch removed them. (See #1503)